virt/creating_vms_advanced/creating_vms_advanced_web/virt-golden-image-heterogeneous-clusters.adoc

```diff
@@ -0,0 +1,37 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="virt-golden-image-heterogeneous-clusters"]
+= Heterogeneous cluster support
+include::_attributes/common-attributes.adoc[]
+:context: virt-creating-vms-from-rh-images-heterogeneous-clusters
+
+toc::[]
+
+:FeatureName: Golden image support for heterogeneous clusters
+include::snippets/technology-preview.adoc[]
+
+A heterogeneous cluster is a cluster where nodes have differing architectures. Heterogeneous clusters promote optimal compute resource usage by mixing different types of hardware in one cluster. This allows workloads to be better matched to hardware intended for the workload task instead of general purpose compute platforms. For example, in a heterogeneous cluster, GPU and general purpose compute resources could be combined and workloads assigned to the appropriate hardware.
+
+[IMPORTANT]
+====
+If golden image support is disabled in a heterogeneous cluster, you can encounter inconsistencies between node and image architectures. This happens when images are used for virtual machine creation that do not match the node architecture. This can lead to the failure of virtual machine scheduling or virtual machines that do not run as expected. The warning level alert `HCOMultiArchGoldenImagesDisabled` is produced when this feature is not enabled in a heterogeneous cluster. 
```

Comment:
One tiny correction:

```suggestion
If golden image support is disabled in a heterogeneous cluster, you can encounter inconsistencies between node and image architectures. This happens when images are used for virtual machine creation that do not match the node architecture. This can lead to the failure of virtual machine boot up or virtual machines that do not run as expected. The warning level alert `HCOMultiArchGoldenImagesDisabled` is produced when this feature is not enabled in a heterogeneous cluster. 
```